rpcserverhelp.go

```diff
@@ -243,8 +243,8 @@ var helpDescsEnUS = map[string]string{
 	"getblockverboseresult-version":           "The block version",
 	"getblockverboseresult-versionHex":        "The block version in hexadecimal",
 	"getblockverboseresult-merkleroot":        "Root hash of the merkle tree",
-	"getblockverboseresult-tx":                "The transaction hashes (only when verbosetx=false)",
-	"getblockverboseresult-rawtx":             "The transactions as JSON objects (only when verbosetx=true)",
+	"getblockverboseresult-tx":                "The transaction hashes (only when verbosity=1)",
+	"getblockverboseresult-rawtx":             "The transactions as JSON objects (only when verbosity=2)",
```

Comment:
2?

Reply:
When `verbosity=2`, the node returns information about each transaction in the block (as opposed to just the hashes of those transactions).

Per the `getblock` command in the [bitcoin RPC docs](https://bitcoincore.org/en/doc/0.19.0/rpc/blockchain/getblock/):

```
If verbosity is 0, returns a string that is serialized, hex-encoded data for block 'hash'.
If verbosity is 1, returns an Object with information about block <hash>.
If verbosity is 2, returns an Object with information about block <hash> and information about each transaction. 
```